Resources/Prototypes/Entities/Objects/Tools/fulton.yml

```diff
@@ -105,3 +105,41 @@
     tags:
       - HideContextMenu
   - type: AnimationPlayer
+
+- type: entity
+  id: ThiefFultonBeacon
+  name: thieving beacon
+  description: A device that will teleport everything around it to the thief's vault at the end of the shift.
+  components:
+    - type: ThiefFulton
```

Comment:
Can this just parent an existing fulton or use composition instead? Looks like it's a bit copied with the normal ones.